compiler/cpp/test/test-files/analyzer/conditional-section-error-boolean-int/input-file.txt

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Marc Auberer 2021. All rights reserved.
```

Comment:
Other test files have a different copyright header.

Reply:
Doesn't matter, as it is only a test file